frontend/lib/norent/letter-builder/tests/confirmation.test.tsx

```diff
@@ -0,0 +1,54 @@
+import { AppTesterPal } from "../../../tests/app-tester-pal";
+import { NorentConfirmation, CANCEL_RENT_PETITION_URL } from "../confirmation";
+import { createProgressStepJSX } from "../../../progress/tests/progress-step-test-util";
+import { override } from "../../../tests/util";
+import { initNationalMetadataForTesting } from "./national-metadata-test-util";
+import { BlankOnboardingInfo } from "../../../queries/OnboardingInfo";
+
+beforeAll(initNationalMetadataForTesting);
+
+describe("NoRent confirmation page", () => {
+  const createPal = (state: string, trackingNumber?: string) => {
+    return new AppTesterPal(createProgressStepJSX(NorentConfirmation), {
+      session: {
+        phoneNumber: "1234567890",
+        onboardingInfo: override(BlankOnboardingInfo, {
+          state,
+        }),
+        norentLatestLetter: trackingNumber
+          ? {
+              trackingNumber: trackingNumber,
+              letterSentAt: null,
+              paymentDate: "Boop 1st, 2099",
+            }
+          : null,
+      },
+    });
```

Comment:
Nice, just FYI i started a "session builder" class to make it easier to construct session objects, especially across different tests.  it's totally optional to use but there if you need it, feel free to add methods to it and stuff if you want: https://github.com/JustFixNYC/tenants2/blob/master/frontend/lib/tests/session-builder.tsx

You can see an example of it in use here:

https://github.com/JustFixNYC/tenants2/pull/1331/files#diff-482a964ea6464f8d2e446ac81e289e44

Totally up to you if you want to use it or not tho!

Reply:
Ok cool! Ok I'll check it out